src/index.js

```diff
@@ -54,7 +54,7 @@ class AttributesKit extends EventEmitter {
   }
 
   render(refractElement) {
-    ReactDom.render(<AttributesComponent element={refractElement} />, this.element);
+    ReactDom.render(<AttributesComponent element={refractElement} dataStructures={this.options.dataStructures} />, this.element);
```

Comment:
Yes, as described in the PR description there is a new `dataStructures` option that, if set, will be used to dereference/resolve/normalize the refract structure. I attempted to make it backward-compatible, so it is optional.